config/app/build.gradle

```
@@ -50,9 +70,20 @@ dependencies {
     // For an optimal experience using Remote Config, add the Firebase SDK
     // for Google Analytics. This is recommended, but not required.
     implementation 'com.google.firebase:firebase-analytics'
+    implementation 'androidx.compose.material3:material3:1.0.0-alpha02'
```

We generally avoid using alpha versions on our quickstarts.
Can we use the stable version of this library instead?
```suggestion
    implementation 'androidx.compose.material3:material3:1.0.1'
```

---

Yep! Stable version works. Will avoid alpha versions moving forward